server/models.py

```diff
@@ -6,11 +6,19 @@
 
 class Lecturer(db.Model):
     id = db.Column(db.Integer, primary_key=True)
+
+    user_name = db.Column(db.String(200), unique=True, nullable=False)
+    password = db.Column(db.String(200), nullable=False)
```

Comment:
It's fine to just add this to the Lecturer Model for now I guess, but should ideally be some common model for both a student and a lecturer (and any eventual other roles), right? E.g. by extending a User class.